_includes/footer.html

```diff
@@ -16,21 +16,22 @@
     </ul>
     <ul>
       <li><h5>Community</h5></li>
-      <li><a href="{{ site.baseurl }}/community/index.html#mailing_lists">Mailing lists</a></li>
-      <li><a href="{{ site.baseurl }}/community/index.html#irc">Social Media</a></li>
+      <li><a href="{{ site.baseurl }}/community/">Community</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#mailing-lists">Mailing Lists</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#chat-rooms">Chat Rooms &amp; More</a></li>
+      <li><a href="{{ site.baseurl }}/community/index.html#community-libraries-and-tools">Libraries and Tools</a></li>
     </ul>
     <ul>
       <li><h5>Contribute</h5></li>
-      <li><a href="{{ site.baseurl }}/contribute/bug-reporting-guide.html">Report an Issue</a></li>
       <li><a href="{{ site.baseurl }}/contribute">How to Help</a></li>
-      <li><a href="{{ site.baseurl }}/contribute/hacker-guide.html">Contributor's Guide</a></li>
-      <li><a href="{{ site.baseurl }}/contribute/scala-fame.html">Contributor's Hall of Fame</a></li>
+      <li><a href="{{ site.baseurl }}/contribute/bug-reporting-guide.html">Report an Issue</a></li>
+      <li><a href="{{ site.baseurl }}/contribute/hacker-guide.html">Hacker's Guide</a></li>
```

Comment:
Actually, I think this renaming came up before which is why in the links it's now called "Contributor's Guide". For people who haven't been to the github repo, they don't know that hacker = contributor, and maybe they would think that the guide is for people who are hardcore Scala hackers and not people who are interested in contributing to the stdlib/compiler. I think someone once made this argument to us which is why we called it "Contributor's guide" in the links here... So maybe it's best that we keep it?